File: backend/src/main/java/by/kapitonov/cinema/backend/service/dto/cinema/UpdateCinemaDTO.java
package by.kapitonov.cinema.backend.service.dto.cinema;

import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.Date;

@Data
@EqualsAndHashCode
public class UpdateCinemaDTO {

    private Long id;
    private String cinemaName;
    private String country;
    private String city;
    private String address;
    private String description;


}

File: frontend/src/app/model/dto/UpdateCinemaDTO.ts
import {Cinema} from "../Cinema";

export class UpdateCinemaDTO {

  id?:number;
  cinemaName?: string;
  country?: string;
  city?: string;
  address?: string;
  description?: string;
  status?: string;

  constructor(cinema: Cinema) {
    this.id = cinema.id;
    this.cinemaName = cinema.cinemaName;
    this.country = cinema.country;
    this.city = cinema.city;
    this.address = cinema.address;
    this.description = cinema.description;
    this.status = cinema.status;
  }

}
